docker-capture-node/default.rules

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Out of curiosity, will we want to make the details here configurable via the CLI?

Reply:
I think eventually we will want folks to be able to add their own rules, and maybe disable these. 

The rulesFile option can take multiple files, my hack solution would be to allow the cli to upload the rules files to s3 and then have the container download them on container start up and check every X minutes for change. Capture can auto reload a changed rules file and doesn't need to restart.

we could also make the rulesFile option take a url and have capture download the new rules instead of something else, but that would take more work.

Reply:
Got it - thanks for the context!